cfme/ansible/repositories.py

```diff
@@ -226,8 +227,11 @@ def create(self, name, url, description=None, scm_credentials=None, scm_branch=N
         repo_list_page = self.create_view(RepositoryAllView)
         assert repo_list_page.is_displayed
         repo_list_page.flash.assert_no_error()
-        repo_list_page.flash.assert_message(
-            'Add of Repository "{}" was successfully initialized.'.format(name))
+        initiated_message = VersionPick({
+            Version.lowest(): 'Add of Repository "{}" was successfully initialized.'.format(name),
+            '5.9.2': 'Add of Repository "{}" was successfully initiated.'.format(name),
+        }).pick(self.browser.product_version)
```

Comment:
**REQUIRED**
```python
if self.appliance.version < "5.9.2":
    initiated_message = 'Add of Repository "{}" was successfully initialized.'.format(name)
else:
    initiated_message = 'Add of Repository "{}" was successfully initiated.'.format(name)
```

Reply:
@quarckster why can't we use VersionPick?

Reply:
Because `VersionPick` is supposed to be used only on views.

Reply:
@quarckster  updated.